.github/actions/node-deploy-cdk/action.yml

```diff
@@ -112,6 +112,19 @@ runs:
       run: npm run diff
       working-directory: ${{ inputs.infrastructure-artifact-name }}
 
+    - name: CDK deploy secrets
+      shell: bash
+      run: npm run deploy-secrets --if-present
+      working-directory: ${{ inputs.infrastructure-artifact-name }}
+
+    - name: Set secrets (secrets)
+      shell: bash
+      run: |
+        if [[ $(jq '.scripts["deploy-secrets"]' < package.json;) != null ]]; then
+          npm run set-secrets --if-present
+        fi
+      working-directory: ${{ inputs.infrastructure-artifact-name }}

```

Comment:
Would we also want to run `npm run set-secrets` below on L153 **only** if there's no `deploy-secrets` npm scripts to avoid running it twice, although admittedly it'll be a no-op?

Reply:
No, because we want it to run secrets after the other stack deployments in case they did have secrets they set.